composer.json

```diff
@@ -18,8 +18,12 @@
         "symfony/property-access":     "~2.1",
         "symfony/expression-language": "~2.4"
     },
+    "suggest": {
+        "twig/twig": "~1.0"
```

Comment:
Maybe `"twig/twig": "Access the state machine in your twig templates (~1.0)"` instead (see https://getcomposer.org/doc/04-schema.md#suggest)